src/cartesian/Line.js

```diff
@@ -28,9 +29,10 @@ class Line extends Component {
     ]), PropTypes.func]),
     unit: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     name: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    dataKey: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
     yAxisId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     xAxisId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    yAxisMap: PropTypes.object,
```

Comment:
@McMenemy  I think this mapping can be done in `LineChart`.  Each `Line` has just one x-axis and one y-axis, It's strange to parse `xAxixMap` and `yAxisMap` to `Line`.